src/gmail.js

```diff
@@ -1655,7 +1655,7 @@ var Gmail_ = function(localJQuery) {
       page = hash;
     }
 
-    if(hash.indexOf('inbox/') !== -1) {
+    if(hash.indexOf('inbox/') !== -1 || hash.indexOf('sent/') !== -1 || hash.indexOf('all/') !== -1) {
       page = 'email';
```

Comment:
Any particular reason this line is changed? Is it required to make undo_send work, or was it just meant as a general fix for another issue?